spec/shr_oncologycore.txt

```diff
@@ -21,23 +21,23 @@ Concept:        SCT#363346000 "Malignant neoplastic disease (disorder)"
 
 EntryElement:   CancerHistologicType
 // did not use precoordinated codes because only available for clinical and pathologic.
-Based on:       SimpleCodedNonLaboratoryObservation
-Description:    "The classification resulting from cancer staging. Each type of cancer has its own Each type of cancer has its own stage groups based on where the cancer has grown and spread. There are either four or five stage groups per cancer. (Source: NCCN Cancer Staging Guide)"
+Based on:       SimpleCodedNonLaboratoryObservation // MK: Would this use a specimen? If so, it has to be a laboratory observation
+Description:    "Classification of the cancer by the type of tissue in which the cancer originates. The international standard for the classification and nomenclature of histologies is the International Classification of Diseases for Oncology, Third Edition (ICD-O-3). (Source: NCI)"
```

Comment:
Remove the second sentence "The international standard for the classification and nomenclature of histologies is the International Classification of Diseases for Oncology, Third Edition (ICD-O-3)" since we are not referencing ICD-O-3 in our value sets.